src/background.js

```diff
@@ -18,120 +21,37 @@ console.log('Settings thirdparty environment variables:', thirdparty_env);
 
 var mainWindow, editWindow, reportWindow;
 
-var setApplicationMenu = function () {
```

Comment:
This whole setup is not good because you can't see how the whole menu is by just looking at the menu file. E.g. a menu file, the menu ends on a separator, and it makes no sense until you read it in this file, but it's not how it should be. Menus should be self contained. that's the reason the menu was here in the first place, and not in any specific file.